SU-SWS/linky_clicky
# BASIC-2344: Add test for deleting content

# READY FOR REVIEW

# Summary
- Tests that bulk delete function in `stanford_manage_content` works as advertised

# Needed By (Date)
- I think whenever a new release is stamped for `stanford_manage_content` and it's added to the products?

# Urgency
- See above, under "Needed By"

# Steps to Test

1. Check out the latest `7.x-3.x-dev` of `stanford_manage_content`
2. Run `includes/features/SWS/stanford_manage_content/stanford_manage_content.feature:37` on a JSA site

# Affected Projects or Products
- All JS* products

# Associated Issues and/or People
- [JIRA ticket](https://stanfordits.atlassian.net/browse/BASIC-2344)
- [PR on stanford_manage_content](https://github.com/SU-SWS/stanford_manage_content/pull/10)
